File: c#/medium_com_rubik.cs
// https://www.codingame.com/training/medium/rubik%C2%AE

using System;

class Solution
{
    static void Main(string[] args)
    {
        int n=int.Parse(Console.ReadLine());
        Console.WriteLine(n>1?6*n*(n-2)+8:1);
    }
}
// Write an action using Console.WriteLine()
// To debug: Console.Error.WriteLine("Debug messages...");

/*
using System.Diagnostics;
class S
{
    static void Main()
    {
        Process.Start(new ProcessStartInfo(
            @"python3","-c\"n=int(input());print(6*n*(n-2)+8 if n>1 else 1)\""));
    }
}
*/

File: c/medium_com_rubik.c
// https://www.codingame.com/training/medium/rubik%C2%AE

#include <stdio.h>
int main()
{
    int n;
    scanf("%d",&n);
    printf("%d\n",n>1?6*n*(n-2)+8:1);
    return 0;
}
// Write an action using printf(). DON'T FORGET THE TRAILING \n
// To debug: fprintf(stderr, "Debug messages...\n");

// main(){system("python3 -c\"n=int(input());print(6*n*(n-2)+8 if n>1 else 1)\"");}
